modules/enabling-scanner-v4-after-operator-installation.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-operator-installation_{context}"]
+= Enabling RHACS Scanner V4 after Operator installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default. If you did not enable it during installation, you can enable it after installation.
+
+First, you must enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. In console, select *Operators* -> *Installed Operators*.
+. In the *Provided APIs* column, click *Central* to view the *stackrox-central-services* page, if you installed {product-title-short} using the default values.
+. Click the name of the cluster where Central was installed. The default value is *stackrox-central-services*.
+. Select the YAML tab.
+. Edit the YAML file as shown in the following example:
++
+[source,yaml]
+----
+  scannerV4:
+    scannerComponent: Enabled
+----
+
+After you enable Scanner V4 on the cluster where Central is installed, you configure your init bundle or cluster registration secret (CRS) to allow communication between Central and the secured cluster. Then, you can enable Scanner V4 on the secured cluster.
+
+.Procedure
+
+. In console, select *Operators* -> *Installed Operators*.
```

Comment:
@dcaravel does this also need to have ` -> *Advanced Cluster Security for Kubernetes*` appending?

Reply:
Looks like it, yes.

Reply:
Yeah, sorry, I spun up a cluster to walk through it myself again and ran out of time. :(  I'll finish up next week and make any changes that need to be made to the language.